### ISSUE TITLE
Data fragmentation and database

### ISSUE THREAD

**Issue by KITblue** [photoprism/photoprism-docs#21]
The number of images generated by the program is too large, fragmentation is serious, and it has a great impact on the hard disk. Do you want to consider using a database

**Comment by lastzero** [comment]
Haven't heard the word fragmentation for a very long time. What file system? 

You typically wouldn't store files in a database. And if you do, the database might as well store them as individual files or suffer from fragmentation if stored in a single binary.

**Comment by lastzero** [comment]
This question on Stackexchange seems related:

https://unix.stackexchange.com/questions/28756/what-is-the-most-high-performance-linux-filesystem-for-storing-a-lot-of-small-fi

**TL;DR: Performance and fragmentation depend a lot on the specific file system**

Money quote:

> The only way to prevent file systems from distributing files over the drive, is to keep the partition only as big as you really need it, but pay attention not to make the partition too small, to prevent intrafile-fragmenting.

**Comment by lastzero** [comment]
I'll close this, since the repo is for our docs only.

If you have a specific feature request or bug report, you're welcome to open an issue in our main repo:

https://github.com/photoprism/photoprism/issues
